### PR TITLE
#KTOR-1027 Update versions

### PR DESCRIPTION
**Pull request by e5l**
Fixes [#KTOR-1027](https://youtrack.jetbrains.com/issue/KTOR-1027)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@
 kotlin.code.style=official
 
 # config
-version=1.4.0
+version=1.4.0-SNAPSHOT
```
Comment (cy6erGn0m) [inline gradle.properties:5]:
```suggestion
version=1.4.1-SNAPSHOT
```

File: gradle.properties [hunk 2]
```diff
@@ -15,14 +15,14 @@ org.gradle.daemon=true
...
 kotlin.native.ignoreDisabledTargets=true
 
 # kotlin libraries
 benchmarks_version=0.2.0-dev-17
 
 serialization_version=1.0.0-RC
-coroutines_version=1.3.8-native-mt-1.4.0-rc
+coroutines_version=1.3.9-native-mt
 atomicfu_version=0.14.4
 validator_version=0.2.2
```
Comment (cy6erGn0m) [inline gradle.properties:27]: 0.2.3?

File: gradle.properties [hunk 3]
```diff
@@ -15,14 +15,14 @@ org.gradle.daemon=true
 org.gradle.jvmargs=-Xmx4g -XX:MaxPermSize=4048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 
 # kotlin
-kotlin_version=1.4.0
+kotlin_version=1.4.10
 kotlin.native.ignoreDisabledTargets=true
 
 # kotlin libraries
 benchmarks_version=0.2.0-dev-17
```
Comment (cy6erGn0m) [inline gradle.properties:22]: -dev-20 is available, we could also try

**Review by cy6erGn0m — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 2 above)
Comment (e5l) [reply gradle.properties:27]: Sure let's update

**Pull request merged**